se_denseslam/CMakeLists.txt

```diff
@@ -52,3 +52,19 @@ target_compile_definitions(${appname}-sdf PUBLIC ${field_type})
 list(APPEND BUILT_LIBS ${appname}-sdf)
 
 set(BUILT_LIBS ${BUILT_LIBS} PARENT_SCOPE)
+
+
+# --------------- INSTALL -----------------------
+INSTALL(
+        TARGETS se-denseslam-ofusion se-denseslam-sdf
+        EXPORT ${appname}Config
+        ARCHIVE DESTINATION /usr/local/lib
+        LIBRARY DESTINATION /usr/local/lib
+        RUNTIME DESTINATION /usr/local/bin
+)
```

Comment:
Remove `/usr/local/`. Also, the RUNTIME DESTINATION shouldn't be needed as there are no executables to install. Although supereight is built as a static library by default, I suggest leaving the LIBRARY DESTINATION to make it easier for anyone who might want to install supereight as a shared library.